src/main/webapp/scripts/common.js

```diff
@@ -24,17 +24,23 @@ window.onscroll = function() {
 
 window.onload = function() {
   fetch("/login").then(result => result.json()).then((details) => {
+    const loginPageURL = window.location.protocol + "//" + window.location.host + "/";
+    console.log(window.location.href);
     if (!details.isUserLoggedIn) {
-        
       // If user is not logged in then redirect to the login page, if not already there
```

Comment:
Please move this comment above the outer `if` as it's describing the whole block